src/main/client/config/common-client-config.ts

```diff
@@ -17,6 +17,7 @@ export class CommonClientConfig extends SharedConfig {
    static readonly MAX_AMMO: number = 10;
    static readonly SCOREBOARD_SIZE: number = 5;
    static readonly MAX_PLAYER_NAME_LENGTH: number = 10;
+   static readonly PLAYER_SPRITE_HEIGHT: number = 48;
```

Comment:
this should be half height, `97` is the height

Reply:
and should I divide by two in the getcenterpoint?

Reply:
Fixed

Reply:
yup divide by 2